docs/tutorial/tasks.md

```diff
@@ -12,7 +12,7 @@ tags:
 
 ## What is a task?
 
-A [task](/concepts/tasks/) is a Python function decorated with a `@task` decorator. Tasks are atomic pieces of work that are executed independently within a flow. Tasks, and the dependencies between them, are displayed in the flow run graph, enabling you to break down a complex flow into something you can observe and understand. When a function becomes a task, it can be executed concurrently and its return value can be cached.
+A [task](/concepts/tasks/) is any Python function decorated with a `@task` decorator called within a flow. You can think of a flow as a recipe for connecting a known sequence of tasks together. Tasks, and the dependencies between them, are displayed in the flow run graph, enabling you to break down a complex flow into something you can observe, understand and control at a more granular level.  When a function becomes a task, it can be executed concurrently and its return value can be cached.
```

Comment:
```suggestion
A [task](/concepts/tasks/) is any Python function decorated with a `@task` decorator called within a flow. 
You can think of a flow as a recipe for connecting a known sequence of tasks together. 
Tasks, and the dependencies between them, are displayed in the flow run graph, enabling you to break down a complex flow into something you can observe, understand and control at a more granular level.  
When a function becomes a task, it can be executed concurrently and its return value can be cached.
```